File: dubbo-service-csharp/trunk/java-hessian-server/src/main/java/com/eqying/pf/service/provider/model/Book.java
package com.eqying.pf.service.provider.model;

import java.io.Serializable;

/**
 * TODO(这个类的作用)
 *
 * @auther: renjunjie
 * @since: 2016/12/12 14:07
 */
public class Book implements Serializable {

	private String id;
	private String name;

	public String getId() {
		return id;
	}

	public void setId(String id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
}

File: dubbo-service-csharp/trunk/dotnet-hessian-client/dotnet-hessian-client/MODEL/Book.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

using System;

namespace com.eqying.pf.service.provider.model
{

    /// <summary>
    /// TODO(这个类的作用)
    /// 
    /// @auther: renjunjie
    /// @since: 2016/12/12 14:07
    /// </summary>
    [Serializable]
    public class Book
    {

        private string id;
        private string name;

        public virtual string Id
        {
            get
            {
                return id;
            }
            set
            {
                this.id = value;
            }
        }


        public virtual string Name
        {
            get
            {
                return name;
            }
            set
            {
                this.name = value;
            }
        }

        public override string ToString()
        {
            return $"{nameof(id)}: {id}, {nameof(name)}: {name}, {nameof(Id)}: {Id}, {nameof(Name)}: {Name}";
        }
    }
}

